### PR TITLE
[next](docs) replace all mentions of `BoxBufferGeometry` with `BoxGeometry`

### PR DESCRIPTION
**Pull request by joshwashywash**

### CODE REVIEW

**Comment by joshwashywash** [comment]
Okay, I guess it was late last night and I wasn’t paying attention but all those BufferGeometry should say BoxGeometry. I’ll fix that later today.

**Comment by joshwashywash** [comment]
ready to go

File: apps/docs/src/content/learn/basics/disposing-objects.mdx [hunk 1]
```diff
@@ -42,7 +42,7 @@ components. This disables disposal for the entire subtree.
 ```svelte
 <script>
   import { T, useTexture } from '@threlte/core'
-  import { BoxBufferGeometry, MeshStandardMaterial } from 'three'
+  import { BoxGeometry, MeshStandardMaterial } from 'three'
```
Comment (michealparks) [inline apps/docs/src/content/learn/basics/disposing-objects.mdx:45]: Looks like we can remove this altogether? (it's just used in a T component)

File: apps/docs/src/content/reference/extras/use-cursor.mdx [hunk 2]
```diff
@@ -32,7 +32,7 @@ Provide arguments to determine the cursor style. The defaults are`'pointer'` for
 
 <T.Mesh
   onpointerenter={onPointerEnter}
-  geometry={new BoxBufferGeometry(1, 1, 1)}
+  geometry={new BoxGeometry(1, 1, 1)}
```
Comment (michealparks) [inline apps/docs/src/content/reference/extras/use-cursor.mdx:35]:
```suggestion
  geometry={new BoxGeometry()}
```

**Review by michealparks — APPROVED** [review]

File: apps/docs/src/content/learn/basics/disposing-objects.mdx (hunk 1 above)
Comment (joshwashywash) [reply apps/docs/src/content/learn/basics/disposing-objects.mdx:45]: yeah we could replace with T.BoxGeometry in each case. same for the material too.

Comment (michealparks) [reply apps/docs/src/content/learn/basics/disposing-objects.mdx:45]: I mean in this case this import is completely unused

Comment (joshwashywash) [reply apps/docs/src/content/learn/basics/disposing-objects.mdx:45]: oh i see okay. i'll get that outta there.

**Pull request merged**